API.md

```diff
@@ -112,8 +140,31 @@
   "survey": "BRES" 
 }
 ```
+#### Example JSON DATA for post Version 2
+* msg_to   Should be set to a specific user uuid if known , else to the constant 'GROUP' if sending to an unknown user in ONS
```

Comment:
consistency on each of these, maybe something like `msg_to - Should be` or `msg_to should be`

Reply:
done